pom.xml

```diff
@@ -134,6 +134,31 @@
             </resource>
         </resources>
 
+        <pluginManagement>
```

Comment:
Why do we need custom call? Can't this just be part of default build process? Like unit tests.